x-pack/plugins/rollup/rollup_data_enricher.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const rollupDataEnricher = async (indicesList, callWithRequest) => {
+  if (!indicesList || !indicesList.length) {
+    return indicesList;
+  }
+  const params = {
+    path: '/_all/_rollup/data',
+    method: 'GET',
+  };
+  const rollupJobData = await callWithRequest('transport.request', params);
```

Comment:
It just allows you to call any arbitrary API endpoint with the path.  I tend to use it just because it documents exactly which endpoint is being used.